tactics/gun-shot-detectors.html

```diff
@@ -43,25 +43,23 @@
     <div class="page-img"><img src="../img/Zapper_4.gif" alt="" style=""></div>
     <div class="page-txt">
         <h1 class="home-head">Gun Shot Detection</h1>
-        
-     <p>Shotspotter is an audio surveillance system. It consists of sensors - microphones - distributed over a neighborhood in order to determine the location of a gunshot. This sensor data can be used to distinguish gunshots from similar sounds such as fireworks and cars backfiring.</p>
+
+     <p>Shotspotter is the largest vendor selling its audio surveillance system. It consists of sensors - microphones - distributed over a neighborhood in order to determine the location of a gunshot. Sensor data is then sent through a set of algorithms and analysts to "verify" a gunshot. Verified alerts are then pushed to police in the field. Shotspotter claims to be able to distinguish gunshots from similar sounds such as fireworks and cars backfiring.</p>
```

Comment:
do you mean "largest vendor of audio surveillance systems"?

Reply:
yep, I can fix